docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,455 @@
++----------------+-------------------------------------------------------------------------------+
+| ratio          | Exchange rate's ratio                                                         |
++----------------+-------------------------------------------------------------------------------+
+| sourceCurrency | :doc:`The currency object serialized with the default data </api/currencies>` |
++----------------+-------------------------------------------------------------------------------+
+| targetCurrency | :doc:`The currency object serialized with the default data </api/currencies>` |
++----------------+-------------------------------------------------------------------------------+
+| updatedAt      | Last update date of the exchange rate                                         |
++----------------+-------------------------------------------------------------------------------+
+
+If you request for more detailed data, you will receive the defualt data with the following additional fields:
+
++----------------+-----------------------------------------+
+| Field          | Description                             |
++================+=========================================+
+| createdAt      | Creation date date of the exchange rate |
++----------------+-----------------------------------------+
+
+Creating an Exchange Rate
+-------------------------
+
+To create a new exchange rate you will need to call the ``/api/v1/exchange-rates/`` endpoint with the ``POST`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/exchange-rates/
+
++----------------+----------------+--------------------------------------+
+| Parameter      | Parameter type | Description                          |
++================+================+======================================+
+| Authorization  | header         | Token received during authentication |
++----------------+----------------+--------------------------------------+
+| ratio          | request        | Ratio of the Exchange Rate           |
++----------------+----------------+--------------------------------------+
+| sourceCurrency | request        | The source currency                  |
++----------------+----------------+--------------------------------------+
+| targetCurrency | request        | The target currency                  |
++----------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.orgg/api/v1/exchange-rates/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
+            {
+                "ratio": "0,8515706",
+                "sourceCurrency": "EUR",
+                "targetCurrency": "GBP"
+            }
+        '
+
+.. tip::
+
+    Remember, that before you will be able to add a new exchange rate both currencies have to be already defined.
```

Comment:
`Remember that` (no comma here), `exchange rate, both currencies` (but here you need one)